src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTable.java

```diff
@@ -380,6 +477,22 @@ else if ( (val instanceof List) && ((List)val).size() == numResultRecords ) {
         }
     }
 
+    /**
+     * Handle per-allele/allele-specific annotations as described in the header
+     * @param val the annotation value
+     * @param result the cummulative output
+     * @param alleleCount scalar, R-type or A-type values
+     */
+    private static void addAlleleSpecificFieldValue(final Object val, final List<List<String>> result, final VCFHeaderLineCount alleleCount) {
```

Comment:
If you have bandwidth/will power, it would be great if you could revamp this legacy code that uses `instanceof` and write separate methods that take parameters of different types.

Reply:
Haha sounds good